osu.Game/Screens/SelectV2/BeatmapMetadataWedge.cs

```diff
@@ -35,6 +35,9 @@ public partial class BeatmapMetadataWedge : VisibilityContainer
         private Drawable failRetryWedge = null!;
         private FailRetryDisplay failRetryDisplay = null!;
 
+        public bool RatingsVisible => ratingsWedge.Alpha > 0;
+        public bool FailRetryVisible => ratingsWedge.Alpha > 0;
```

Comment:
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Reply:
test fails like shit if you actually got this code right

Reply:
force-pushed fix in https://github.com/ppy/osu/pull/32957/commits/bc1bf68e79f5fec03ecb3a762976017701de422f, tests don't actually fail.